turbopack/crates/turbo-tasks-backend/src/backend/operation/aggregation_update.rs

```diff
@@ -51,71 +54,89 @@ fn get_followers_with_aggregation_number(
     }
 }
 
+/// Returns a list of tasks that are considered as "following" the task. The current tasks is not
+/// aggregating over the follower tasks and they should be aggregated by all upper tasks.
 fn get_followers(task: &impl TaskGuard) -> Vec<TaskId> {
     get_followers_with_aggregation_number(task, get_aggregation_number(task))
 }
 
+/// Returns a list of tasks that are considered as "upper" tasks of the task. The upper tasks are
+/// aggregating over the task.
 pub fn get_uppers(task: &impl TaskGuard) -> Vec<TaskId> {
     get_many!(task, Upper { task } count if *count > 0 => *task)
 }
 
+/// Returns an iterator of tasks that are considered as "upper" tasks of the task. See `get_uppers`
 fn iter_uppers<'a>(task: &'a (impl TaskGuard + 'a)) -> impl Iterator<Item = TaskId> + 'a {
     iter_many!(task, Upper { task } count if *count > 0 => *task)
 }
 
+/// Returns the aggregation number of the task.
 pub fn get_aggregation_number(task: &impl TaskGuard) -> u32 {
     get!(task, AggregationNumber)
         .map(|a| a.effective)
         .unwrap_or_default()
 }
 
+/// A job in the job queue for updating something in the aggregated graph.
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub enum AggregationUpdateJob {
+    /// Update the aggregation number of a task. This might return in balancing needed to update
+    /// "upper" and "follower" edges.
```

Comment:
```suggestion
    /// Update the aggregation number of a task. This might result in balancing needed to update
    /// "upper" and "follower" edges.
```
is that what you meant?